components/furnituremix/application/src/app/core/components/payments/crystal/coin.tsx

```diff
@@ -20,7 +22,7 @@ export const CrystalCoin: React.FC = () => {
             disabled={paying}
             onClick={async () => {
                 setPaying(true);
-                await ServiceAPI.sendPaidOrderWithCrystalCoin(cart, customer);
+                await ServiceAPI(state.locale, state.serviceApiUrl).sendPaidOrderWithCrystalCoin(cart, customer);
```

Comment:
Should we just store the service api instance in context? Its a bit strange creating an instance so many places and having to pass config.

Reply:
it's per request, not sure we can do that